### ISSUE TITLE
Low AUC results with pre-trained weights of UCF-Crime

### ISSUE THREAD

**Issue by JonathanFlores2503** [AnasEmad11/C2FPL#5]
Thank you for sharing your excellent work. I encountered issues replicating your results using the UCF-Crime dataset. 

I used `Concat_test_10.npy` for test features, `gt-ucf-RTFM.npy` for ground truth, and all pre-trained weights in `./ckpt/`, but I couldn't achieve the results reported in the paper.

Could you advise on how to achieve the shared results or provide the specific weights used? 
Below, you can see the table with the obtained results.

| Pretrained Weights | AUC |
| ------ | ------ |
| UCFfinal     | 0.7695754476190766 |
| deepmil1-i3d | 0.7832887135425388 |
| deepmil2-i3d | 0.5398925011234721 |
| deepmil3-i3d | 0.625350145373792  |
| deepmil4-i3d | 0.5877377170247249 |
| deepmil5-i3d | 0.763993071397966  |
| deepmil6-i3d | 0.7855268017804337 |
| deepmil7-i3d | 0.7033726733129118 |
| deepmil8-i3d | 0.7491821113223371 |
| deepmil10-i3d | 0.7520102514259424 |
| deepmil12-i3d | 0.6922479689283714 |

**Comment by jpainam** [comment]
The released code is inconsistent. Maybe that's not the final version. You are also not updating the pseudo-labels each epoch.

What the use of `new_labels` in `train.py`?,

You also defined `self.fc_att1` and `self.fc_att2` in `model.py,` but you don't use them in `Model_V2 nn.Module`.
Can you confirm if that is the final code that produced the reported results?

Thank you.

**Comment by KirinHao** [comment]
Hello, have you solved this problem? I have the same question, thank you!

**Comment by jpainam** [comment]
I did not solve  the problem. My best performance using the provided weight is also `0.769`

**Comment by mohdwarish483** [comment]
@jpainam  @KirinHao have u solved this problem and do u have training script for feature extraction,I am not getting that how to train on my own data

**Comment by WuTao-maker** [comment]
@jpainam @KirinHao @mohdwarish483 do u have training script for feature extraction？,I cannot train my own data too，could you help me?

**Comment by jpainam** [comment]
you can download the feature from https://github.com/tianyu0207/RTFM. see the readme
